scripts/build_components.sh

```diff
@@ -37,20 +39,40 @@ for dir in "$component_dir"/*/; do
 
   BASENAME=${dir%/}
   BASENAME=${BASENAME##*/}
-  full_image_name=ghcr.io/${namespace}/${BASENAME}:${tag}
-  echo "$full_image_name"
 
-  # Update the image version in the fondant_component.yaml
-  sed -i "s|^image: .*|image: $full_image_name|" fondant_component.yaml
+  echo "Tagging image with following tags:"
+  for tag in "${tags[@]}"; do
+    full_image_name=ghcr.io/${namespace}/${BASENAME}:${tag}
+    echo "$full_image_name"
+    full_image_names+=("$full_image_name")
+  done
+  # Prevent this from mistakenly being used below
+  unset full_image_name
 
-  # Update the fondant requirement to the version being built
-  sed -i "s|fondant@main|fondant@$tag|" requirements.txt
+  echo "Updating the image version in the fondant_component.yaml with:"
+  echo "${full_image_names[0]}"
+  sed -i "s|^image: .*|image: ${full_image_names[0]}|" fondant_component.yaml
```

Comment:
So here you update the images, how does this make it into main ?

Reply:
It doesn't. This actually doesn't have an effect when triggered on main, only when releasing, as the updated component specifications are included in the `fondant` package. You could argue that it shouldn't be in this script, but then we need to create the `full_image_name`s in multiple places.

Come to think of it, I do think we have an issue in the release pipeline now, as this script only runs when triggered by a tag, not by a release. So the component specs are only updated in the package pushed to `test.PyPi`, not to the actual `PyPi`.

Reply:
Opened #255 for this.

Reply:
Ok makes sense, then this PR is a nice upgrade